Tests/StreamChatUITests/SnapshotTests/ChatChannelList/Search/ChatChannelSearchVC_Tests.swift

```diff
@@ -74,16 +74,17 @@ final class ChatChannelSearchVC_Tests: XCTestCase {
 
         vc.currentSearchText = "Some message"
         vc.executeLifecycleMethods()
+        vc.channels = mockedChannelListController.channels_mock ?? []
```

Comment:
I know it is nice to ease tests, but channels should be private(set) at least

Reply:
This was actually not for tests, since now the data source of the channel list is the `channels` property, I don't think that `private(set)` is a good option here, just like it is not the case for the `channelListController`. Customers might want to update the data from another view as well, we don't know 🤔 I don't there is any benefit of making it private(set) but let me know if you have any other scenario in mind 👍

Reply:
I don't think it is a good approach to allow others to mutate the source of truth of our views.
At most, I would have a method that explicitly mentions what it does (and internally calls reloadData or whatever is needed)

Reply:
A good thing about making it public is that it could allow customers to override the DataSource and inject additional channels. We already had a couple of requests like this before in the message list for example this one: https://github.com/GetStream/stream-chat-swift/issues/2305

Either way, I'll make it private(set) for now, and if we customers asking for this, then we can open it up

Reply:
@polqf Done ✅